modules/development/pages/overview.adoc

```diff
@@ -0,0 +1,9 @@
+= Overview
+
+This module will guide a user through the installation of the Fawkes project using a local development environment.
+
+This entails:
+
+* Setting up a host machine or VM to act as the Hypervisor
```

Comment:
```suggestion
* Set up a hypervisor host
```

"A host machine" sounds funny, and since a machine/workstation and a VM are both _hosts_ we can make this less wordy.

Reply:
Updated